CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contribute to netprobify
+
+## Requirements
+
+Pull requests require the following:
+- unit tests
+- documentation via DocStrings, and update README.md if needed
+- coding style respected
+
+## Environment
+
+Please see the README.md file. It contains everything needed to run the tests and build the project.
+
+## When my PR will be reviewed and merged?
+
+Usually, we expect reviews to be done under one month of delay.
+
+However, the delay might increase if there is too much pending PR or big features being developped internally.
+
+Once the code has been reviewed and validated, it will be merged into the master branch.
+
+## Version release
+
+Each version will be tested under stable environment.
```

Comment:
Did you mean "in a test environment" ?

Reply:
nope. I meant it will tested internally on our side (meaning in our network)

Reply:
is it clearer now?